.github/ISSUE_TEMPLATE/02-chaosscast-episode-checklist.md

```diff
@@ -0,0 +1,89 @@
+
+### Scheduling an episode
+
+* **Step 1:** Propose a topic and guest(s) \(Anyone\)
+  * [ ] **Action:** Discuss with facilitator to get approval for the episode topic and guest.
+  * [ ] **Action:** Recruit interview guest to finalize topic and content of the podcast together
+* **Step 2:** Facilitate Episode \(Facilitator\)
+  * [ ] **Action:** Send invitation email to guest to find a few potential suitable dates and times \(60-90 minutes.\)
+  * [ ] **Action:** Notify organizer of guest's date and time possibilities, and confirm one.
+  * [ ] **Action:** Prepare the planning document (copy the [Episode Planning Document template](https://docs.google.com/document/d/17qjAafr0aAWXV_HR-WiefHOa881t8X7tEoiZnPBh3ao/edit?pli=1#heading=h.cmpmu92ylw4d) and start filling in information.)
+  * [ ] **Action:** Invite panelists from the CHAOSS community who have expertise in the episode topic.
+  * [ ] **Action:** Confirm panelists 
+  * [ ] **Action:** Send calendar invitation with confirmed date of the podcast 
+    * The event has to be created in the [CHAOSScast calendar](https://calendar.google.com/calendar/ical/513r47bb756t07lp5tat5nsuj0%40group.calendar.google.com/public/basic.ics)
+      * All organizers should have full access to the calendar \(i.e., Make changes and manage sharing\)
+    * Name calendar event:🎙  CHAOSScast {Topic}
+      * Start with the microphone emoticon \(🎙 \) to signal recording meeting
+  * [ ] **Action:** Send out a [confirmation email](https://docs.google.com/document/d/1Pw8OKn-qSSkMw573BZu-EZ4BZualesdZjPbpBEKvzrc/edit) that includes the [“how we podcast” document](https://docs.google.com/document/d/1j5Ng91rrFrx3PdPejv26rBxb1P7UX-O_Q0_I9TCIya0/edit#) to guests and panelists
+  * [ ] **Action:** Update the episode planning document with recording time and zoom link
+  * [ ] **Action:** Update the [episode overview table](https://docs.google.com/document/d/1qxmC1k8SayChurIq2r91CR3r_KAThTwaNEMjA6DLGhE/edit#)
+  * [ ] **Action:** If the episode is more than two weeks out, schedule a [reminder email](https://docs.google.com/document/d/17vKzCOYvRXq5zaIlv_sb80zFapTU19dRgkx5RqRx3xE/edit) for 1 week before the episode recording
+  * [ ] **Action:** Schedule a [reminder email](https://docs.google.com/document/d/17vKzCOYvRXq5zaIlv_sb80zFapTU19dRgkx5RqRx3xE/edit) for 24-hours before the episode recording
+* **Step 3:** Finalize Content \(Organizer\)
+  * [ ] **Action:** Update the episode planning document with the main topics and questions
+* **Step 4:** Post-Episode Thank You Notes \(Georg Link\)
+  * [ ] **Action:** When the episode is scheduled for release that week, notify guest and get their mailing address
+  * [ ] **Action:** Sign thank you note and assemble with swag
+  * [ ] **Action:** Mail out thank you note with swag
+  * [ ] **Action:** Add date of shipment on [episode overview table](https://docs.google.com/document/d/1qxmC1k8SayChurIq2r91CR3r_KAThTwaNEMjA6DLGhE/edit) to confirm it was sent
```

Comment:
You can probably take this step off. I was not great about keeping this updated, tbh, and it's an unnecessary step. I started it mostly as a way to keep track for international purposes when things were sent.

Reply:
Agreed to take this off